framework/widgets/ActiveField.php

```diff
@@ -748,4 +758,14 @@ protected function getClientOptions()
             'error' => '.help-block',
         ]);
     }
+
+    /**
+     * Returns the HTML `id` of the input element of this form field.
+     * @return string the input id.
+     * @since 2.0.6
```

Comment:
Thank you! https://github.com/yiisoft/yii2/commit/723e3a9850d6175cffcc5013e1a5f6b5baca4c61